presto-main-base/src/main/java/com/facebook/presto/security/AccessControlManager.java

```diff
@@ -68,30 +69,34 @@
 import static com.google.common.collect.ImmutableSet.toImmutableSet;
 import static java.lang.String.format;
 import static java.util.Objects.requireNonNull;
+import static org.weakref.jmx.ObjectNames.generatedNameOf;
 
 public class AccessControlManager
         implements AccessControl
 {
     private static final Logger log = Logger.get(AccessControlManager.class);
     private static final File ACCESS_CONTROL_CONFIGURATION = new File("etc/access-control.properties");
     private static final String ACCESS_CONTROL_PROPERTY_NAME = "access-control.name";
+    private static final String ACCESS_CONTROL_DETAILED_STATS = "access-control.detailed-stats";
```

Comment:
Is this a property in the plugin?  If so, why not make this a property of the runtime, i.e. a Presto property?  And I would simply make this export all the time, we usually don't need to gate metrics.